src/Serilog.Sinks.InMemory/InMemorySink.cs

```diff
@@ -3,12 +3,14 @@
 using System.Threading;
 using Serilog.Core;
 using Serilog.Events;
+using Serilog.Formatting;
 
 namespace Serilog.Sinks.InMemory
 {
     public class InMemorySink : ILogEventSink, IDisposable
     {
         private static readonly AsyncLocal<InMemorySink> LocalInstance = new AsyncLocal<InMemorySink>();
+        private readonly ITextFormatter _formatProvider;
```

Comment:
Given that the `_formatProvider` is never used I would recommend removing it, that would also remove the necessity for `GetInstance()` as you can use `Instance` like before.